tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenWaitForPrimaryChange.cs

```diff
@@ -0,0 +1,111 @@
+﻿/* Copyright 2020–present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.Misc;
+using MongoDB.Driver.Core.Servers;
+
+namespace MongoDB.Driver.Tests.JsonDrivenTests
+{
+    public sealed class JsonDrivenWaitForPrimaryChange : JsonDrivenTestRunnerTest
```

Comment:
this test fails from time to time. According to the latest discussions, it will be probably a server ticket